engine/deeplearning/src/main/java/org/opennms/alec/engine/deeplearning/InputVector.java

```diff
@@ -1,8 +1,8 @@
 /*******************************************************************************
  * This file is part of OpenNMS(R).
  *
- * Copyright (C) 2019 The OpenNMS Group, Inc.
- * OpenNMS(R) is Copyright (C) 1999-2019 The OpenNMS Group, Inc.
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
```

Comment:
these files were added in 2019, this should be 2019-2022 then

Reply:
To be clear, If I update a file from 2019 in 2022, should I update the copyright from 2019 to 2019-2022 ?  

* Copyright (C) 2019-2022 The OpenNMS Group, Inc.
 * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.